stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -79,7 +79,20 @@ else() # NOT SWIFT_BUILT_STANDALONE
   set(clang_headers_location "${LLVM_LIBRARY_DIR}/clang/${CLANG_VERSION}")
 endif()
 
-add_custom_command_target(unused_var
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+  add_custom_command_target(unused_var
+    COMMAND
+      "${CMAKE_COMMAND}" "-E" "make_directory" "${SWIFTLIB_DIR}"
+    COMMAND
+      "mklink" "/J"
```

Comment:
Hmm,  have an account with elevated privileges. I've noticed that I get privilege errors using `mklink` Without any arguments. They seem to go away when I add `/H` or `/J`.

That said, i can change to be a basic copy_if_different/copy if you'd like.